.github/workflows/codecov.yml

```diff
@@ -23,7 +24,7 @@ jobs:
         # Credentials would need to be configured.
         git submodule init
         git submodule update
-        python -m pytest -v --disable-warnings --cov=./ --cov-report=xml:coverage.xml tests
+        python -m pytest -v --disable-warnings --cov=./ --cov-report=xml:coverage.xml tests dsgrid/utils/py_expression_eval/tests.py
```

Comment:
What is `dsgrid/utils/py_expression_eval/tests.py` about? Also, I don't see that file in this PR, does it already exist?